slices/apache2.yaml

```diff
@@ -0,0 +1,319 @@
+      /etc/apache2/mods-enabled/rewrite.load:
+        symlink: /etc/apache2/mods-available/rewrite.load
+      /etc/apache2/mods-enabled/sed.load:
+        symlink: /etc/apache2/mods-available/sed.load
+      /etc/apache2/mods-enabled/session.load:
+        symlink: /etc/apache2/mods-available/session.load
+      /etc/apache2/mods-enabled/session_cookie.load:
+        symlink: /etc/apache2/mods-available/session_cookie.load
+      /etc/apache2/mods-enabled/session_crypto.load:
+        symlink: /etc/apache2/mods-available/session_crypto.load
+      /etc/apache2/mods-enabled/session_dbd.load:
+        symlink: /etc/apache2/mods-available/session_dbd.load
+      /etc/apache2/mods-enabled/setenvif.conf:
+        symlink: /etc/apache2/mods-available/setenvif.conf
+      /etc/apache2/mods-enabled/setenvif.load:
+        symlink: /etc/apache2/mods-available/setenvif.load
+      /etc/apache2/mods-enabled/slotmem_plain.load:
+        symlink: /etc/apache2/mods-available/slotmem_plain.load
+      /etc/apache2/mods-enabled/slotmem_shm.load:
+        symlink: /etc/apache2/mods-available/slotmem_shm.load
+      /etc/apache2/mods-enabled/socache_dbm.load:
+        symlink: /etc/apache2/mods-available/socache_dbm.load
+      /etc/apache2/mods-enabled/socache_memcache.load:
+        symlink: /etc/apache2/mods-available/socache_memcache.load
+      /etc/apache2/mods-enabled/socache_redis.load:
+        symlink: /etc/apache2/mods-available/socache_redis.load
+      /etc/apache2/mods-enabled/socache_shmcb.load:
+        symlink: /etc/apache2/mods-available/socache_shmcb.load
+      /etc/apache2/mods-enabled/speling.load:
+        symlink: /etc/apache2/mods-available/speling.load
+      /etc/apache2/mods-enabled/ssl.conf:
+        symlink: /etc/apache2/mods-available/ssl.conf
+      /etc/apache2/mods-enabled/ssl.load:
+        symlink: /etc/apache2/mods-available/ssl.load
+      /etc/apache2/mods-enabled/status.conf:
+        symlink: /etc/apache2/mods-available/status.conf
+      /etc/apache2/mods-enabled/status.load:
+        symlink: /etc/apache2/mods-available/status.load
+      /etc/apache2/mods-enabled/substitute.load:
+        symlink: /etc/apache2/mods-available/substitute.load
+      /etc/apache2/mods-enabled/suexec.load:
+        symlink: /etc/apache2/mods-available/suexec.load
+      /etc/apache2/mods-enabled/unique_id.load:
+        symlink: /etc/apache2/mods-available/unique_id.load
+      /etc/apache2/mods-enabled/userdir.conf:
+        symlink: /etc/apache2/mods-available/userdir.conf
+      /etc/apache2/mods-enabled/userdir.load:
+        symlink: /etc/apache2/mods-available/userdir.load
+      /etc/apache2/mods-enabled/usertrack.load:
+        symlink: /etc/apache2/mods-available/usertrack.load
+      /etc/apache2/mods-enabled/vhost_alias.load:
+        symlink: /etc/apache2/mods-available/vhost_alias.load
+      /etc/apache2/mods-enabled/xml2enc.load:
+        symlink: /etc/apache2/mods-available/xml2enc.load
```

Comment:
This list of `mods-enabled` symlinks that we are creating seems incorrect.
The only modules that the `postinst` script are enabling is the following:
```
                a2enmod -m -q mpm_event

                for module in authz_host auth_basic access_compat authn_file authz_user \
                                alias dir autoindex \
                                env mime negotiation setenvif \
                                filter deflate \
                                status reqtimeout ; do
                        a2enmod -m -q $module
                done
```
Can you review and shorten this list to just those specific modules?